dependencies.toml

```diff
@@ -479,10 +479,7 @@ version.ref = "krotodc"
 [libraries.guava]
 module = "com.google.guava:guava"
 version.ref = "guava"
-exclusions = [
-    "com.google.errorprone:error_prone_annotations",
-    "com.google.j2objc:j2objc-annotations",
```

Comment:
We cannot exclude these dependencies anymore:
[https://github.com/google/guava/commit/4787cf5591646d25918edc7af29b9f7c1ed4034b#diff-af93ada7ef59cd024b38a39e3f0[…]83e4d6ff77595e3e2879831cddR54-R61](https://github.com/google/guava/commit/4787cf5591646d25918edc7af29b9f7c1ed4034b#diff-af93ada7ef59cd024b38a39e3f0763b91b4b1483e4d6ff77595e3e2879831cddR54-R61)